test_driver/mock_helper.dart

```diff
@@ -42,4 +44,80 @@ class MockHelper {
           HttpHeaders.authorizationHeader: 'Bearer ${Data.ACCESS_TOKEN}'
         })).thenAnswer((_) async => http.Response(Data.SECOND_PAGE, 200));
   }
+
+  // Prepares the response to the mocked client when it calls for the
```

Comment:
Just wondering why sometimes you use "///" and other times "//"

Reply:
'///' is for doc comment and '//' is a simple comment.